src/jit/morph.cpp

```diff
@@ -14051,11 +14051,21 @@ GenTree* Compiler::fgMorphModToSubMulDiv(GenTreeOp* tree)
     {
         numerator = fgMakeMultiUse(&tree->gtOp1);
     }
+    else if (lvaLocalVarRefCounted && numerator->OperIsLocal())
+    {
+        // Morphing may introduce new lclVar references. Increase ref counts
+        lvaRecursiveIncRefCounts(numerator);
```

Comment:
I think that the new calls to `lvaIRecursiveIncRefCounts` can just be `lvaIncRefCnts` rather than the recursive version. Also, the comment is a bit stale, as we have explicitly introduced a new lclVar reference.